src/controllers/handlers/livekit-webhook-handler.ts

```diff
@@ -0,0 +1,91 @@
+import { DecentralandSignatureContext } from '@dcl/platform-crypto-middleware'
+import { HandlerContextWithPath } from '../../types'
+import { WebhookReceiver } from 'livekit-server-sdk'
+import { IHttpServerComponent } from '@well-known-components/interfaces'
+
+enum WebhookEvent {
+  ParticipantJoined = 'participant_joined',
+  ParticipantLeft = 'participant_left'
+}
+
+const TOPIC_SUFFIX_BY_EVENT = {
+  [WebhookEvent.ParticipantJoined]: 'join',
+  [WebhookEvent.ParticipantLeft]: 'leave'
+}
+
+function isValidEvent(event: string): event is keyof typeof WebhookEvent {
+  return ['participant_joined', 'participant_left'].includes(event)
+}
```

Comment:
What are the advantages of using this function contract? wouldn't be easier do it like this?:

```suggestion
function isValidEvent(eventName: string): boolean {
  return ['participant_joined', 'participant_left'].includes(eventName)
}
```

And maybe also use a type for *participant_joined* and *participant_left*

Reply:
I'm using the event received by parameter and need to verify that it matches one of the enum values before using it in a dictionary that accepts only valid events.